firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/SessionReportingCoordinator.java

```diff
@@ -130,6 +132,27 @@ public void persistNonFatalEvent(Throwable event, Thread thread, long timestamp)
     persistEvent(event, thread, EVENT_TYPE_LOGGED, timestamp, false);
   }
 
+  public void finalizeNativeEvent(
```

Comment:
Minor stylistic request: Can we call this `finalizeSessionWithNativeEvent`, passing the session ID and the native files?

Reply:
sure